src/server.rs

```diff
@@ -845,16 +937,15 @@ fn main() {
                             }
                         } else {
                             let err = decode_result.unwrap_err();
-                            println!("ERROR decoding packet from {:?}: {}", addr, err.description());
+                            if !server_state.silence_error {
+                                println!("ERROR decoding packet from {:?}: {:?}", addr, err);
+                            }
+                            server_state.silence_error = false;
```

Comment:
Why are we turning off error silencing here?

Reply:
We've design this pathway to return a `ResponseCode`. For the case where we get a duplicate packet (checked via sequence number), we should just ignore it and not do anything. But because of this pathway, we need to return something so I returned an `AlreadyExists` error. KeepAlive sends a packet with the latest sequence number so it will always trip the `AlreadyExists` case every time. I do not want to print an error statement when this happens.

I was considering returning `ResponseCode::Ok`, but I wanted to avoid sending back traffic in this case as its unnecessary IMO which is why I return a specific error and send nothing.